components/charts/chartsjs/BarChart.js

```diff
@@ -14,7 +14,8 @@ const calculateData = (recordKeys, records) => {
   const filterItems = (arr, query) => arr.filter(record => record === query);
   // Calculate the total # of deaths per data type
   // if value is null return 0 otherwise return total # of deaths for this data type
-  const deathsByDataType = recordKeys.map(key => (!key ? 0 : filterItems(records, key).length));
+
+  const deathsByDataType = recordKeys.sort((a, b) => a - b).map(key => (!key ? 0 : filterItems(records, key).length));
```

Comment:
I think this is the default behavior of `sort`, so we could do this:

```suggestion
  const deathsByDataType = recordKeys.sort().map(key => (!key ? 0 : filterItems(records, key).length));
```

There's also a lot going on in this line in general, so I'd be open to two other approaches as well:
1. Handle the `recordKeys.sort()` on its own line above. I was a little confused when I saw that we were sorting `recordKeys` for `deathByDataType` but not below where we return the `labels`, but on further investigation, I saw that `sort` mutates the original array. Handling that on its own line might make that more clear.
2. Allow the caller of `<BarChart>` to handle the sorting. I think https://github.com/texas-justice-initiative/website-nextjs/pull/73 should do that, so we might actually already be fine here.

Reply:
Thanks @nholden . This is a good point about how `sort` mutates the original array. `map` returns a new array, so I think combining them in one line makes this doubly confusing. I'm going to separate it out to make things more clear.